ansible_collections/arista/avd/roles/eos_cli_config_gen/templates/eos/end.j2

```diff
@@ -0,0 +1,5 @@
+{# hostname #}
```

Comment:
Should this be end?

Reply:
haha no. Thank you for spotting this. Too much copy/paste.